rules/windows/command_and_control_certutil_network_connection.toml

```diff
@@ -2,9 +2,35 @@
 creation_date = "2020/03/19"
 integration = ["endpoint", "windows"]
 maturity = "production"
-updated_date = "2023/01/31"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
+updated_date = "2023/02/27"
+
+[transform]
+[[transform.osquery]]
+label = "Osquery - Retrieve DNS Cache"
```

Comment:
Nit: I think we can just keep the label as the name since `transform.osquery` is already defined. Seems a little redundant. Thoughts?

<img width="867" alt="Screenshot 2023-03-06 at 1 02 41 PM" src="https://user-images.githubusercontent.com/99630311/223194088-a907388f-b2a4-4dee-831f-2fbe5cd8444e.png">

Reply:
I am not sure if I am completely understanding. Do you mean that rather than

```
label = "label ..."
query = "query ..."
```

we just do

```
<label_name> = <actual_query>
```

If so, the problem become handling keys of unknown nature

Reply:
Nope my question is do we need `Osquery - ` in the label since it is already defined as an Osquery in `transform.osquery`. I would assume that because these are transform fields, it is good to keep the `Osquery - ` as that will be the label in the markdown?

Reply:
Ah, makes more sense. I think it better to literally translate than to insert osquery into that text. So IMO it is more of a rule consistency question.